js/tiles.js

```diff
@@ -518,6 +518,24 @@ visit: function() {
       this.wallRight.show();
       this.ground.show();
   }
+}),
+highLow: Object.assign({}, OpenDoors, {
```

Comment:
I mentioned these lines. Most lines are uintended, but it's okay as we are looking to provide a good experience in game.

Reply:
What is exactly the problem with this line?

Reply:
Add two spaces to the left end as it is an object of the door class.

Reply:
There is no such indentation on the above objects as well. I will make a new issue for indentation and work on it. It wouldn't be good to just randomly change the indentation of one block in the middle of the file.

Reply:
I don't know if that issue will be feasable as it would lead to conlficts however it is upto you. As i mentioned for this PR it looks good to be merged.